wowup-electron/src/assets/i18n/en.json

```diff
@@ -310,6 +310,7 @@
         "ADDONS_SELECTED": "{count} {count, plural, =1{addon} other{addons}} selected",
         "ALPHA_ADDON_CHANNEL": "Alpha",
         "AUTO_UPDATE_ADDON_BUTTON": "Auto Update",
+        "IGNORE_AUTO_UPDATE_ADDON_NOTIFICATIONS_BUTTON": "Ignore Notifications",
```

Comment:
Do an ```npm run i18n``` to have it sync a value to all locales and alphabatize it for you from EN

Reply:
TIL, cheers!